tests/test_stories.py

```diff
@@ -56,10 +53,68 @@
       'story_name': 'Time for Hack4Impact',
     }
 ]
-            
-          
-    story = {
-    '_id': 25
+
+
+story_empty = {

```

Comment:
i think the name `story_empty` might be confusing since it could imply that it's an invalid story with an incomplete payload. but since it looks like its a story without any pois, it could be clearer to call it something like `story_with_no_pois`. it'll also have a parallel with the other story called `story_with_pois`